src/runtime/client-config-manager.js

```diff
@@ -30,9 +30,10 @@ export class ClientConfigManager {
   /**
    * @param {string} publicationId
    * @param {!./fetcher.Fetcher} fetcher
+   * @param {!./deps.DepsDef} deps
    * @param {!../api/basic-subscriptions.ClientOptions=} clientOptions
    */
-  constructor(publicationId, fetcher, clientOptions) {
+  constructor(publicationId, fetcher, deps, clientOptions) {
```

Comment:
Can you please move deps to the first argument? We're not consistent, but I'd like to make the managers put deps in first in all cases. Since it's required, you'll need to update all the usages anyways.

Reply:
Makes sense. Done